PyInstaller/building/toc_conversion.py

```diff
@@ -132,7 +132,11 @@ def __collect_data_files_from_zip(zipfilename):
         # 'PyInstaller.config' cannot be imported as other top-level modules.
         from ..config import CONF
         workpath = os.path.join(CONF['workpath'], os.path.basename(zipfilename))
-        os.makedirs(workpath)
+        try:
+            os.makedirs(workpath)
+        except OSError as e:
+            if e.errno != 17:
```

Comment:
*shiver* Please use symbolic names, not dubious constant values. This is what the [manual](https://docs.python.org/2/library/exceptions.html#exceptions.OSError) recomments, too. `errno.EEXITS` ought to be the one you want to use here.